alembic/versions/873e7cedb671_.py

```diff
@@ -53,15 +55,15 @@ def upgrade() -> None:
     op.create_table('cart',
     sa.Column('id', sa.String(), nullable=False),
     sa.Column('user_id', sa.String(), nullable=False),
-    sa.Column('cart_total_price', sa.Float(), nullable=False),
+    sa.Column('cart_total_price', sa.DECIMAL(), nullable=False),
     sa.ForeignKeyConstraint(['user_id'], ['user.id'], onupdate='cascade', ondelete='cascade'),
     sa.PrimaryKeyConstraint('id'),
     sa.UniqueConstraint('id')
     )
     op.create_table('category_product_association_table',
     sa.Column('category_id', sa.String(), nullable=False),
     sa.Column('product_id', sa.String(), nullable=False),
-    sa.ForeignKeyConstraint(['category_id'], ['category.id'], onupdate='cascade', ondelete='cascade'),
+    sa.ForeignKeyConstraint(['category_id'], ['category.id'], onupdate='cascade', ondelete='SET NULL'),
```

Comment:
part of the next pull request where order history with removed products will be kept - products will stay even when category is removed, so category_id value in product-category M2M table should be enabled to be null